### ISSUE TITLE
Localize “email address” placeholder on home page

### ISSUE THREAD

**Issue by sftim** [kubernetes/website#19272]
**This is a Feature Request**

**What would you like to be added**
On the page https://k8s.io/ there is a [form](https://kubernetes.io/#mc_embed_signup) that lets you sign up to receive KubeWeekly. That form has placeholder text “email address”, that is currently always English.

**Why is this needed**
The website should be localized, including placeholder text.

**Comments**
Here's how to make this change:
1. Edit [`i18n/config.toml`](https://github.com/kubernetes/website/blob/master/i18n/en.toml) to add a new section:
    > ```toml
    > [input_placeholder_email_address]
    > other = "email address"
    >
    > ```
2. Edit [`layouts/index.html`](https://github.com/kubernetes/website/blob/b12bde0845e35de76731cf213177a3b5e5eb418a/layouts/index.html) to replace:
    > ```html
    >         	<input type="email" value="" name="EMAIL" class="email" id="mce-EMAIL" placeholder="email address" aria-label="email" required>
    > ```
    with updated HTML that uses the `T` shortcode for localization. Have a look at other shortcodes in that page for an idea of how this works.
3. [Submit a pull request](https://kubernetes.io/docs/contribute/start/#improve-existing-content).
4. Once your pull request is merged, log a new issue for each localization (see https://github.com/kubernetes/website/tree/master/i18n for a list) to get each localization team to add and translate `input_placeholder_email_address`. Label each issue for the matching localization language code, eg `/language ko` for the Korean team.
5. That's it!

**Note** You could use a different name from `input_placeholder_email_address` if you want to.

/help
/kind cleanup
/priority backlog

**Comment by sftim** [comment]
I haven't labelled this as a “good first issue” but if you'd like to tackle this as your first issue, feel free. 

For help, you can the [Kubernetes Slack](http://slack.k8s.io/) and ask questions in the [#sig-docs](https://kubernetes.slack.com/messages/C1J0BPD2M/) channel. It's also fine to reply here.

**Comment by sftim** [comment]
It's localized!
/close
